demoapps/simplest/src/main/res/layout/activity_create_account.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
Sorry I forgot about this on the first pass. We should mark the root view as not important for Autofill.  Same for the login layout.

Reply:
Fixed